docs/ref/xref.md

```
@@ -523,6 +527,7 @@
 [TLS transport]: ../tran/tls.md
 [udp]: ../tran/udp.md
 [streams]: ../api/stream.md
+[id map]: ../api/id_map.md
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "id map"

(MD053, link-image-reference-definitions)